src/arch/aarch64/ram64.s

```diff
@@ -0,0 +1,32 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright (C) 2022 Akira Moroo */
+
+.section .text.boot, "ax"
+.global ram64_start
+
+ram64_start:
+  /* UEFI "MZ" signature magic instruction */
+  add x13, x18, #0x16   /* code0 */
+  b jump_to_rust        /* code1 */
+
+  .quad 0               /* text_offset */
+  .quad 0               /* image_size */
+  .quad 0               /* flags */
+  .quad 0               /* res2 */
+  .quad 0               /* res3 */
+  .quad 0               /* res4 */
+
+  .long 0x644d5241      /* "ARM\x64" magic number */
```

Comment:
Doesn't this line mean the magic number of "ARMD"? Where doesn't this come from? Can you give a source link here?

Reply:
Please read https://github.com/cloud-hypervisor/rust-hypervisor-firmware/pull/205#issuecomment-1327301458.